### DISCUSSION TITLE
blog/2024/02/15/PolarCTF-Reverse-Easy-Writeup/

### DISCUSSION THREAD

**Discussion by giscus** [L0serQianXia/blog#3] [Announcements]
# blog/2024/02/15/PolarCTF-Reverse-Easy-Writeup/

https://l0serqianxia.github.io/blog/2024/02/15/PolarCTF-Reverse-Easy-Writeup/

**Comment by L0serQianXia** [comment]
图片没有懒加载，卡爆了

**Comment by L0serQianXia** [comment]
图片有懒加载了，还是卡爆了

**Comment by L0serQianXia** [comment]
更完啦好耶

**Comment by giraffexiu** [comment]
动调大师，我全是静态秒的

**Reply by L0serQianXia** [reply]
差不多（
想着算法不是很难就全都看反汇编过一遍了
上错号了寄

**Reply by giraffexiu** [reply]
哈哈哈，你也是大一的吗，你的github邮箱上锁了o(*////▽////*)q，想找你聊天的

**Reply by L0serQianXia** [reply]
之前有什么奇怪的国外公司在发远程工作的邮件，被骚扰了好几次就关掉邮箱显示了诶

**Reply by L0serQianXia** [reply]
目前是高中在读，虽然做题的时候想记录一下过程就有了博客，但是真要想尝试把他写明白真的好累（

**Reply by L0serQianXia** [reply]
这些题第一次做的时候也是拿ida直接出的伪代码，借这次机会也是顺便读读反汇编，毕竟有些题会让伪代码显示得不准确，polarctf的reverse里好像还没有这种题

**Reply by giraffexiu** [reply]
三分逆向七分猜，高中厉害的，我大一才刚学，可以来我博客指点指点www.giraffexiu.love，加个qq聊聊
